perceval/backends/core/git.py

```diff
@@ -968,7 +968,7 @@ def rev_list(self, branches=None):
         if branches is None:
             cmd_rev_list.extend(['--branches', '--tags', '--remotes=origin'])
         elif len(branches) == 0:
-            cmd_rev_list.append('--max-count=0')
+            cmd_rev_list.extend(['--branches', '--tags', '--max-count=0'])
```

Comment:
If `branches` is an empty list, it should return an empty list of references, shouldn't it?

Reply:
I agree with you, for consistency with the codebase I mimicked the code at: https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/git.py#L1030. Should that part also be changed?

Reply:
Sorry, I understood wrongly the patch. I get now why that's needed. To run the command. :)